plugins/woocommerce/src/Blocks/Shipping/ShippingController.php

```diff
@@ -129,16 +129,40 @@ public function show_local_pickup_details( $return_value, $order ) {
 		$details         = $shipping_method->get_meta( 'pickup_details' );
 		$location        = $shipping_method->get_meta( 'pickup_location' );
 		$address         = $shipping_method->get_meta( 'pickup_address' );
+		$cost            = $shipping_method->get_total();
 
-		if ( ! $address ) {
+		$section = '';
```

Comment:
This might be cleaner if we avoid adding `<br/>` to the beginning of the strings below and instead add each item to an array. Then we can implode with `<br/>` later and return the result.